_includes/components/footer-navigation.html

```diff
@@ -1,13 +1,12 @@
 {% assign logo = include.logo | default: 'logo.svg' %}
 {% assign logo_width = include.logo_width | default: 191 %}
 {% assign logo_height = include.logo_height | default: 27 %}
-{% assign logo_alt = include.logo_alt | default: 'login.gov' %}
```

Comment:
This footer partial is also used for the Partners site, so dropping the handling of `include.logo_alt` means that the partner site logo will change from "login.gov partners" to "login.gov logo". 

https://github.com/18F/identity-site/blob/9a0178610b2b5bb830b63e4278c82b8aacab6f6f/_includes/partners/footer.html#L2

I'd suggest either:

* Keeping this behavior and assigning the `default` from the new `login_gov_logo` string data
* Or removing the value from where it's being passed in if we want to have this change also apply to the partners site
   * I don't know that we do want to make this change. Ideally we'd also be updating the partner logo text alternative to "login.gov partners logo", but maybe we can go with the first option and file a ticket for the partnerships team to tackle the partner logo themselves.